components/financial_assistance/app/models/financial_assistance/applicant.rb

```diff
@@ -1320,10 +1321,11 @@ def set_evidence_attested(evidence)
       save!
     end
 
-    def set_evidence_outstanding(evidence)
+    def set_evidence_outstanding(evidence, due_date = nil)
       return unless evidence.may_move_to_outstanding?
 
       evidence.verification_outstanding = true
+      evidence.due_on = due_date if evidence.due_on.blank?
```

Comment:
Modify `due_date` to `desired_due_date `

what do you think about modifying this to `evidence.due_on = desired_due_date if desired_due_date.present?` just so when some future usecase arrives, this method supports it.